database/DashboardSchema.js

```diff
@@ -0,0 +1,12 @@
+const mongoose = require('mongoose')
+const Schema = mongoose.Schema
+const WidgetSchema = require('./WidgetSchema')
+
+const DashboardSchema = mongoose.Schema ({
+  _user: {type: Number, ref: 'User'},
```

Comment:
Shouldn't this be an `ObjectID`?  Or is this some functionality provided by mongoose?  Why is the name `_user` instead of `user`?